### ISSUE TITLE
edit link and segmentation

### ISSUE THREAD

**Issue by mxardre** [wiggins-lab/SuperSegger#2]
Hi,
I am playing with supperSegger and most of the time it work like a charm. However sometimes there are some errors in the linkage or the segmentation. Would it be possible to edit the links in the supperseggerViewer?
All the best
Maxime
And thanks again for this nice tool!

**Comment by stellastyl** [comment]
Hi Maxime, 

we don't have a way to edit the links unfortunatelly - it's in the todo list to create a tool to edit the links, but I am not sure when we will get to it. 
Theoretically if you fix the wrong segments manually and re-link it should get the links right. If you notice a specific way that the linking fails, I could look a bit into our linking code and see if I can fix it.

Best,
Stella

**Comment by mxardre** [comment]
Unfortunatly I think it is not a segmentation problem because bacteria are well segmented. If i understood how things work, I think the errors come from the fact that the bacteria are moving too much from on frame to another.
I would much appreciate your idea on this, and if I can help I would be very glad. Shall I send you some sample?
All the best
Maxime

**Comment by stellastyl** [comment]
Oh okay, it is will try to link them the best they can if they are moving too much, but it may not always get it right. Is your frame rate quite high? An experimental way to fix this is to take more phase images if possible, and keep the current frame rate for fluorescence images if that's a requirement.
I would be happy to look at the images. You can send me a couple of the images that have issues, and the _err files for those images so that I can see what the linking looks like.

**Comment by mxardre** [comment]
Hi Stella,

Here is a sample of what I have done :

https://dl.espci.fr/ticket/84306120e7e1e9947a259872799610e9
https://dl.espci.fr/ticket/1d72c0d67b586a8e6039cd30f532a917
https://dl.espci.fr/ticket/bf184d79f6072da1b155f77fd896936d

You can get back the file by doing in a terminal : cat
testSegger.tar.gz.aa testSegger.tar.gz.ab testSegger.tar.gz.ac | tar xvf -

And I used the homemade training set attached.

As you can see there is some error but not much on the segmentation.
Maybe a should retrain my settings?

Maxime

-- 
Dr Maxime Ardré

Laboratoire génétique de l'évolution.
École supérieure de physique et de chimie industrielles de la ville de Paris
10 Rue Vauquelin, 75005 Paris

Tel : 01 40 79 44 42

**Comment by stellastyl** [comment]
Hi Maxime, downloading these and unpacking them is quite slow. Could you just send 4-5 frame files (the _err.mat files that are in the /seg folder)? Just choose the frame numbers where the problem is visible.
Thanks!

**Comment by mxardre** [comment]
Hi Stella,

Sorry for the late answer. I had to put this project in standby a bit,
but now am in again.
I am currently analysing datas. I realized that a was maybe expecting to
much from the software. Few errors will produce outlayer that I can
handle after while.
Because I do not need the lineage of the colonies it should be all right.

I have made a home made training set. It seems to work more or less so I
am happy with that.

Thank you very much for your help.

Keep in touch

Maxime



-- 
Dr Maxime Ardré

Laboratoire génétique de l'évolution.
École supérieure de physique et de chimie industrielles de la ville de Paris
10 Rue Vauquelin, 75005 Paris

Tel : 01 40 79 44 42
